torch_np/_normalizations.py

```diff
@@ -104,7 +105,46 @@ def normalize_this(arg, parm, return_on_failure=_sentinel):
         return arg
 
 
-def normalizer(_func=None, *, return_on_failure=_sentinel):
+# postprocess return values
+
+
+def postprocess_ndarray(result):
+    return _helpers.array_from(result)
+
+
+def postprocess_out(result):
+    result, out = result
+    kwds = postprocess_out.kwds
+    return _helpers.result_or_out(result, out, **kwds)
+
+
+def postprocess_tuple(result):
+    return _helpers.tuple_arrays_from(result)
+
+
+def postprocess_list(result):
+    return list(_helpers.tuple_arrays_from(result))
+
+
+def postprocess_variadic(result):
+    # a variadic return: a single NDArray or tuple/list of NDArrays, e.g. atleast_1d
+    if isinstance(result, (tuple, list)):
+        seq = type(result)
+        return seq(_helpers.tuple_arrays_from(result))
+    else:
+        return _helpers.array_from(result)
+
+
+postprocessors = {
+    NDArray: postprocess_ndarray,
+    OutArray: postprocess_out,
```

Comment:
As discussed, I think we should be able to implement the `out=` keyword generically. Let's catch-up on this today.